src/main/resources/jenkins/plugins/openstack/compute/OpenStack.groovy

```diff
@@ -0,0 +1,97 @@
+package jenkins.plugins.openstack.compute
+
+import jenkins.plugins.openstack.compute.internal.DestroyMachine
+import jenkins.plugins.openstack.compute.internal.Openstack
+import org.openstack4j.model.compute.Server
+
+class OpenStack implements Serializable {
+
+    private org.jenkinsci.plugins.workflow.cps.CpsScript script
+
+    public OpenStack(org.jenkinsci.plugins.workflow.cps.CpsScript script) {
+            this.script = script
+    }
+
+    public Node server(String cloudName, String template) {
+        new Node(this, cloudName, template)
```

Comment:
So this whole DSL is a waste of effort. Would be better written as a single `Step`, much more simply add with no Groovy or class loading tricks. The return value could be a POJO.